src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1867,7 +1874,7 @@ public final boolean isAlive() {
      * This method is non-final so it can be overridden.
      */
     boolean alive() {
-        return isAlive0();
+        return eetop != 0;
```

Comment:
For consistency, this should use `NO_REF` instead of `0`.
Please remove the method `isAlive0()` that is no longer used.

Reply:
Moved `isAlive0()` implementation into `alive()`.